### ISSUE TITLE
ancient module not really needed

### ISSUE THREAD

**Issue by craff** [craff/gles3#8]
ancient module is only needed by example spheres ... which should not need it anymore when multicore is out ...

**Comment by rlepigre** [comment]
Indeed, for some reason pinning the repository fails if ancient is not installed, although I think it should not since the dependency is marked as optional (and so is the example using ancient in its dune file). No idea why we get that behaviour.

**Comment by craff** [comment]
Done! Use ocaml 5 domain instead of processes
